tabsie/server.R

```diff
@@ -14,11 +14,11 @@ shinyServer(
     #The data package now reads in:
       #serverData - a list of data frames. Data Frame one should be the default. All data frames
                   ## should have the SAME columns. They should merely be subsets of the original.
-      #serverDataDic - A list of definitions ofr the filters: E.g. c("Not filtered", "Filtered)
+      #serverDataDic - A list of definitions ofr the filters: E.g. c("Not filtered", "Filtered")
       #serverHash - the hash function for the server authentication. If you don't want an 
                   ## authentication screen make this = ""
     #REPLACED loading functions
-        load("survSave.rdata")
+    if(file.exists('survSave.rdata')) load("survSave.rdata")
```

Comment:
Put into a conditional statement so that if the file doesn't exist, instead of erroring, TABSIE just does the same thing it would do with an invalid survSave.rdata